engine/src/button.h

```diff
@@ -123,7 +123,9 @@ class MCButton : public MCControl
 	uint1 accelmods;
 	uint1 mnemonic;
 	uint1 menucontrol;
-    MCGravity m_icon_gravity : 4;
+	// SN-2015-01-06: [[ Bug 14306 ]] The type of an enum is implementation-defined,
```

Comment:
Probably worth correcting the comment here slightly - state that MCGravity has 12 members, enums are signed so you need 5 bits in a bit field.